travis_ci_bootstrap.sh

```diff
@@ -0,0 +1,18 @@
+if [[ ${TRAVIS_OS_NAME} == "osx" ]]
+then
+    # download and install EDM
+    wget https://package-data.enthought.com/edm/osx_x86_64/1.3/edm_1.3.0.pkg
+    sudo installer -pkg edm_1.3.0.pkg -target /
+else
+    # set up XVFB for GUI tests
+    sh -e /etc/init.d/xvfb start
```

Comment:
I would still keep this out of the bootstrap script. In some cases it is not necessary and I have seen cases one might want to adjust the parameters.